00-intro/intro.md

```diff
@@ -15,39 +15,33 @@ links-as-notes: true
 
 ## Overview
 
-\Gls{NeoFS} is a decentralized distributed object storage system integrated with
-the [Neo Blockchain](https://neo.org).
+\Gls{NeoFS} is a decentralized distributed object storage system integrated with [Neo Blockchain](https://neo.org).
 
-\Glspl{Node} are organized in peer-to-peer network that takes care of storing
-and distributing user's data. Any Neo user may participate in the network and
-get paid for providing storage resources to other users or store his data in
-NeoFS and pay a competitive price for it.
+We store and distribute users' data across a peer-to-peer network of \Glspl{Node}. Whether a business or an individual, any Neo user may join the network and get paid for providing storage resources to others or pay a competitive price and enjoy NeoFS storage solution.
 
-Users can reliably store object data in the NeoFS network and have a transparent
-data placement process due to decentralized architecture and flexible storage
-policies. Under the users we mean both individuals and enterprises. Each
-\Gls{Node} is responsible for executing the storage policies that the users
-select for geographical location, reliability level, number of nodes, type of
-disks, capacity, etc. Thus, NeoFS gives full control over data to users.
+The decentralized architecture and flexible storage
+policies allow users to reliably store object data in the NeoFS network. Each \Gls{Node} is responsible for executing the storage policies selected by the users
+regarding the geographical location, reliability level, number of nodes, type of
```

Comment:
To avoid word repetition, maybe we should change `reliability level` to `redundancy level` here.

Reply:
I don't feel like redundancy level means the same as reliability level... In this case I'd rather change reliably store to safely store?

Reply:
Reliability is more general term, while redundancy is on of the possible ways to achieve a certain level of system's reliability. In this case we set how many independent object replicas must be put into the storage network. Those replicas are called redundant data copies and the number of copies is often called redundancy level.

Reply:
Now I see tht it's what you provide. In this case, I agree. Thanks for the explanation :)